packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,192 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Content toolbar
+
+### Standard view
+
+```js
+<Toolbar
+	contentKind="Video"
+	showClose={true}
+	title="Counting with small numbers"
+/>
+```
+
+### Content completed view
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+const startExercise = <Button>Next Video</Button>;
+
+<Toolbar
+	contentKind="Video"
+	rightContent={[startExercise]}
+	showClose={false}
+	title="Counting with small numbers"
+/>
+```
+
+## Exercise toolbar
+
+### Generic header
+
+```js
+const Link = require("@khanacademy/wonder-blocks-link").default;
+
+// TODO(scottgrant): This is a placeholder icon. The icon in Zeplin was rotated.
+const caretPath = "M18.629 15.997l-7.083-7.081L13.462 7l8.997 8.997L13.457 25l-1.916-1.916z";
+const caretIcon = (
+    <svg role="img" width="24px" height="24px">
+        <path fill="#1865f2" d={caretPath} />
+    </svg>
+);
+//const goToExercise = <Link>Go to exercise {caretIcon}</Link>;
+const goToExercise = <a href="#">Go to exercise</a>;
+
+<Toolbar
+	rightContent={[goToExercise]}
+	showClose={true}
+	subtitle="0 of 7 questions answered"
+	title="Exercise Title"
+/>
+```
+
+### Start of task card
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+const questionCount = <span>7 questions</span>;
+const startExercise = <Button>Start exercise</Button>;
+
+<Toolbar
+	rightContent={[questionCount, startExercise]}
+	showClose={false}
+/>
+```
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+const nextExercise = <Button>Next exercise</Button>;
+const questionCount = <span>7 questions</span>;
+const tryAgain = <Button kind="secondary">Try again</Button>;
+
+<Toolbar
+	rightContent={[questionCount, tryAgain, nextExercise]}
```

Comment:
This is an important example b/c it shows off the ordering of elements.  I like this example better than the previous one b/c it has more buttons.  Since they're so similar let's drop the previous one.

Reply:
👍